qt/widgets/instrumentview/src/InstrumentWidgetPickTab.cpp

```diff
@@ -51,6 +53,17 @@ namespace MantidWidgets {
 
 using namespace boost::math;
 
+namespace {
+double getPhi(const Mantid::Kernel::V3D &pos) {
```

Comment:
Comments required. Implies beam along x, y vertical? pos is offset from sample, or just works because sample is at 0,0,0?

Reply:
Same bellow with offset.